lib/reporting/protocols_report.rb

```diff
@@ -181,7 +181,7 @@ def saml_signature_query
     def cloudwatch_client
       @cloudwatch_client ||= Reporting::CloudwatchClient.new(
         num_threads: @threads,
-        ensure_complete_logs: true,
+        ensure_complete_logs: false,
```

Comment:
the queries in this file uses `| stats` or `| dedup` so I think the "split over time and combine" approach is not applicable here